### PR TITLE
Remove UNS Integration

### PR DESCRIPTION
**Pull request by mplorentz**
## Issues covered
#1636

## How to test
- Verify that UNS UI is absent from the profile and search results screens.
- Search the codebase for references to Universal Names. 

## Screenshots/Video
Post screenshots or video showing your changes, ideally showing how the app worked before and after these changes. Delete this section if this PR contains no visual changes.

| Before | After |
|--------|--------|
|Simulator Screenshot - iPhone 16 for real - 2024-10-15 at 16 18 51|Simulator Screenshot - iPhone 16 for real - 2024-10-15 at 16 27 39|

### CODE REVIEW

File: Nos/Views/Components/Form/NosTextEditor.swift [hunk 1]
```diff
@@ -11,6 +11,7 @@ struct NosTextEditor: View {
             TextEditor(text: $text)
                 .textInputAutocapitalization(.none)
                 .foregroundColor(.primaryTxt)
+                .scrollContentBackground(.hidden)
```
Comment (mplorentz) [inline Nos/Views/Components/Form/NosTextEditor.swift:14]: I noticed this bug a few days ago but hadn't filed a ticket yet. I decided to slip the fix in here. 🙈

Comment (mplorentz) [reply Nos/Views/Components/Form/NosTextEditor.swift:14]: You can see the bug-  a black background for the bio field - in the `before` screenshot in the PR description.

**Comment by joshuatbrown** [comment]
👀

File: Nos/Views/Components/Form/NosTextEditor.swift (hunk 1 above)
Comment (joshuatbrown) [reply Nos/Views/Components/Form/NosTextEditor.swift:14]: Oh I saw this bug, too, so I'm glad this is being fixed.

File: Nos/Models/RawNostrID.swift [hunk 2]
```diff
@@ -20,7 +20,11 @@ public typealias RawEventID = RawNostrID
 /// An alias for a RawNostrID that we know is for an Author. See docs for `RawNostrID`.
 public typealias RawAuthorID = RawNostrID
 
-extension RawNostrID {
+extension RawNostrID: @retroactive Identifiable {
+    
+    public var id: String {
```
[the remark this answers is not in the document]
Comment (joshuatbrown) [reply Nos/Models/RawNostrID.swift:25]: 🤷

**Comment by pelumy** [comment]
110 files 😲, just to remove a view? That view must have been pretty important. This is an insane amount of work.

**Review by joshuatbrown — COMMENTED** [review]
This looks good. I think there's more. I'll target this branch with a new PR so you can confirm.

**Comment by mplorentz** [comment]
> 110 files 😲, just to remove a view? That view must have been pretty important. This is an insane amount of work.

@pelumy well, it wasn't really just one view. There was actually a wizard with a bunch of steps and then we displayed the names in several other places like search results and the profile screen.

**Review by joshuatbrown — APPROVED** [review]
Yes! Always love to see -5K lines of code in a PR!

**Pull request merged**